packages/consumption/test/modules/requests/DecideRequestParametersValidator.test.ts

```diff
@@ -143,7 +120,7 @@ describe("DecideRequestParametersValidator", function () {
 
     const errorParams: TestParam[] = [
         {
-            description: "(1) error: Request with two items is answered with one item",
+            description: "(1) error: Request with two RequestItems is answered with one item",
```

Comment:
Request with two RequestItems is answered with one **ResponseItem**

Reply:
I struggle with this formulation because, strictly speaking, you do not respond with a ResponseItem, but with certain DecideRequestItemParameters, from which ResponseItems are then generated if the Request is successfully answered. Nevertheless, I do not like this wording either, so if you have an alternative suggestion that avoids the problem I mentioned, please let me know.